spec/models/chat_channel_membership_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:chat_channel_membership) { FactoryBot.create(:chat_channel_membership) }
 
   describe "#channel_text" do
-    it "sets channel text using name, slig, and human names" do
+    it "sets channel text using name, slug, and human names" do
```

Comment:
This really doesn't have to do with fixing the bug, but since it's just a quick typo fix related to the same subject of `ChatChannelMembership` I figured I'd just include it here.